templates/template/docs/validation.md

```diff
@@ -0,0 +1,47 @@
+## Repository Validation / CI
+
+In order to provide a way for CI to ensure that the current state of a repository is valid and all necessary commits that produce generated changes have been run by developers, `make validate` runs a series of checks on a clean Git repository.
+
+Specifically, the workflow used by `make validate` does the following:
+1. Ensure Git is clean; if not, fail.
+2. Run `make charts`; if Git is no longer clean, fail and leave behind the assets.
+3. **Only if `validate.url` and `validate.branch` are provided in the `configuration.yaml`**, pull in the specified Git repository, standardize the repository, and check each asset:
+  - For any assets that exist in upstream, check if it is modified or does not exist in local. If so, copy it over, unzip it, and fail.
+  - For any assets that exist in local but not in upstream, check if it corresponds to an entry in the `release.yaml`; if not, fail.
+4. Run `make unzip`; if Git is no longer clean, fail.
+
+### What is the release.yaml?
```

Comment:
Is it true that we need to wipe out the `release.yaml` file after releasing those new charts, i.e., merging the PR from dev-v2.6 to release-v2.6?

Reply:
Yes, we do need to wipe it out. It would be similar to leaving behind `doNotRelease` on a chart, which basically means there's a gap in our CI as a result.

Reply:
We should add it to the list of post-release actions ( if we have a such list 😆 )

Reply:
I don't think such a list exists, but we can add it to the release captain notes after the first release with these scripts.